src/sentry/options/defaults.py

```diff
@@ -3655,3 +3655,10 @@
     default=0.0,
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
+
+register(
+    "seer.scanner_no_consent.rollout_rate",
```

Comment:
nit but i believe we prefer to use hyphens rather than underscores in options

Reply:
ah yeah, that's not surprising. kind of a convention that's hard to stick to/enforce. we document it [here](https://develop.sentry.dev/backend/application-domains/options/) but i couldn't tell you why 🤷‍♂️